Tools/Frame_params/eLAB_LAB445_AC34.param

```diff
@@ -2,28 +2,69 @@
 ACRO_YAW_P,3
 ANGLE_MAX,3500
 ARMING_CHECK,-10
-BATT_AMP_PERVOLT,39.877
+ATC_ACCEL_P_MAX,95000.91
+ATC_ACCEL_R_MAX,100000.7
+ATC_ACCEL_Y_MAX,27000
+ATC_ANG_LIM_TC,1
+ATC_ANG_PIT_P,18
+ATC_ANG_RLL_P,15.1205
+ATC_ANG_YAW_P,4.5
+ATC_ANGLE_BOOST,1
+ATC_RAT_PIT_D,0.007277923
+ATC_RAT_PIT_FF,0
+ATC_RAT_PIT_FILT,20
+ATC_RAT_PIT_I,0.09861624
+ATC_RAT_PIT_IMAX,0.5
+ATC_RAT_PIT_P,0.09861624
+ATC_RAT_RLL_D,0.005631518
+ATC_RAT_RLL_FILT,20
+ATC_RAT_RLL_I,0.09034792
+ATC_RAT_RLL_IMAX,0.5
+ATC_RAT_RLL_P,0.09034792
+ATC_RAT_YAW_I,0.018
+ATC_RAT_YAW_P,0.18
+ATC_SLEW_YAW,6000
+BATT_AMP_PERVOLT,17
 BATT_CAPACITY,3500
+BATT_CURR_PIN,3
 BATT_MONITOR,3
-BATT_VOLT_MULT,10.2
+BATT_VOLT_MULT,10
+BATT_VOLT_PIN,2
 BRD_SAFETYENABLE,0
 CH7_OPT,3
 CH8_OPT,4
 COMPASS_EXTERNAL,1
 COMPASS_USE3,0
 FENCE_ALT_MAX,50
 FENCE_RADIUS,150
+FLTMODE1,0
+FLTMODE2,0
+FLTMODE3,0
 FLTMODE4,2
 FLTMODE5,3
 FLTMODE6,5
 FRAME_CLASS,1
-FRAME,1
+FRAME_TYPE,1
+FS_BATT_ENABLE,2
+FS_BATT_MAH,0
 FS_BATT_VOLTAGE,10.7
-FS_THR_VALUE,990
+FS_THR_ENABLE,1
+FS_THR_VALUE,975
 GND_ABS_PRESS,100475.7
 GND_EFFECT_COMP,1
+GPS_TYPE,2
 LOG_BITMASK,397310
 MOT_THST_HOVER,0.2557016
+MOT_SPIN_ARM,0.1
+MOT_SPIN_MIN,0.12
+RTL_ALT,1500
+SERIAL0_BAUD,115
+SERIAL0_PROTOCOL,1
+SERIAL1_BAUD,38
```

Comment:
My telemetry has a baud rate of 57,600.
https://store.shopping.yahoo.co.jp/elab-store/ld013.html